Frontend/src/components/usermanagement/SettingsProfilePic.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <div>
+    <v-row class="row1">
+      <v-col cols="4">
+        <p>{{ $t(changeprofilepic) }} <br> {{ $t(advice) }}</p>
+      </v-col>
+      <v-col col="4" class="test">
+        <v-list-item>
+          <v-list-item-avatar>
+            <v-img :src="profilePicture"></v-img>
+          </v-list-item-avatar>
+        </v-list-item>
+      </v-col>
+      <v-col cols="5">
+        <v-file-input
+            label="profile_picture"
```

Comment:
Label in etwas aussagekräftiges umbennen, da es im Frontend beim User angezeigt wird. 
Am besten direkt auch mit Localisierung.
Vorschlag: Neues Profilbild / New Profilepicture